docs/package.json

```diff
@@ -27,6 +27,7 @@
     "@emotion/server": "11.11.0",
     "@emotion/styled": "11.11.0",
     "@mui/icons-material": "5.15.6",
+    "@mui/joy": "5.0.0-beta.24",
```

Comment:
Is this needed for any of the changes in this PR? Or was it missing before?

Reply:
It seemed to be missing before